crates/oxc_linter/src/rules/unicorn/prefer_array_some.rs

```diff
@@ -46,7 +46,7 @@ declare_oxc_lint!(
     /// ```
     PreferArraySome,
     pedantic,
-    pending
+    fix
```

Comment:
Should this be a `conditional_fix`? I see a `fixer.noop` in this diff.

Reply:
`fixer.noop` should never be hit, it's just there to avoid panics in the event something very very strange has happened